packages/css/src/grid/page-grid.scss

```diff
@@ -0,0 +1,19 @@
+/**
+ * @license EUPL-1.2+
+ * Copyright (c) 2023 Gemeente Amsterdam
+ */
+
+@import "../../utils/breakpoint";
+
+.amsterdam-page-grid {
+  column-gap: 2%;
```

Comment:
I like the last one with em's but clamp could prevent the gap from "exploding" when people have their font size maxed out.

I came across this article, it might help 

https://moderncss.dev/contextual-spacing-for-intrinsic-web-design/#gap